aspnetcore/blazor/host-and-deploy/webassembly.md

```diff
@@ -35,32 +35,32 @@ Blazor relies on the host to the serve the appropriate compressed files. When us
 * For IIS `web.config` compression configuration, see the [IIS: Brotli and Gzip compression](#brotli-and-gzip-compression) section. 
 * When hosting on static hosting solutions that don't support statically-compressed file content negotiation, such as GitHub Pages, consider configuring the app to fetch and decode Brotli compressed files:
 
-  * Reference the Brotli decoder from the [google/brotli GitHub repository](https://github.com/google/brotli/) in the app.
+  * Obtain the JavaScript Brotli decoder from the [google/brotli GitHub repository](https://github.com/google/brotli/). As of July, 2020, the decoder file is named `decode.min.js` and found in the repository's `js` folder.
   * Update the app to use the decoder. Change the markup inside the the closing `<body>` tag in `wwwroot/index.html` to the following:
   
     ```html
-    <script src="brotli.decode.min.js"></script>
+    <script src="decode.min.js"></script>
     <script src="_framework/blazor.webassembly.js" autostart="false"></script>
     <script>
-    Blazor.start({
-      loadBootResource: function (type, name, defaultUri, integrity) {
-        if (type !== 'dotnetjs' && location.hostname !== 'localhost') {
-          return (async function () {
-            const response = await fetch(defaultUri + '.br', { cache: 'no-cache' });
-            if (!response.ok) {
-              throw new Error(response.statusText);
-            }
-            const originalResponseBuffer = await response.arrayBuffer();
-            const originalResponseArray = new Int8Array(originalResponseBuffer);
-            const decompressedResponseArray = BrotliDecode(originalResponseArray);
-            const contentType = type === 
-              'dotnetwasm' ? 'application/wasm' : 'application/octet-stream';
-            return new Response(decompressedResponseArray, 
-              { headers: { 'content-type': contentType } });
-          })();
+      Blazor.start({
+        loadBootResource: function (type, name, defaultUri, integrity) {
+          if (type !== 'dotnetjs' && location.hostname !== 'localhost') {
+            return (async function () {
+              const response = await fetch(defaultUri + '.br', { cache: 'no-cache' });
+              if (!response.ok) {
+                throw new Error(response.statusText);
+              }
+              const originalResponseBuffer = await response.arrayBuffer();
+              const originalResponseArray = new Int8Array(originalResponseBuffer);
+              const decompressedResponseArray = BrotliDecode(originalResponseArray);
+              const contentType = type === 
+                'dotnetwasm' ? 'application/wasm' : 'application/octet-stream';
+              return new Response(decompressedResponseArray, 
+                { headers: { 'content-type': contentType } });
+            })();
```

Comment:
What function is being invoked here? The `loadBootResource` one? I don't think the invocation is needed...

Reply:
@pranavkm ... I just remembered that you supplied the content for this on https://github.com/dotnet/AspNetCore.Docs/pull/18697. Can you help?

Reply:
This one: https://github.com/dotnet/AspNetCore.Docs/pull/19335/files#diff-3dab49620f61444e10e46b0f178bb1bbR48.

Reply:
OH! I see. I assume we do this because we don't support an async `loadBootResource` method.

Thanks for helping me spot this.

Reply:
Cool ..... everything else ok? This one can go in immediately if so.